include/z64camera.h

```diff
@@ -6,7 +6,7 @@
 #include "z64save.h"
 
 // these two angle conversion macros are slightly inaccurate
-#define CAM_DEG_TO_BINANG(degrees) (s16)((degrees) * 182.04167f + .5f)
+#define CAM_DEG_TO_BINANG(degrees) (s16)TRUNCF_BINANG((degrees) * 182.04167f + .5f)
```

Comment:
Probably worth adding `#include z64math.h` in the camera header, pretty sure this only works because while `z64camera.h` comes above `z64math.h` in `z64.h`, `z64math.h` is included in headers above `z64camera.h` (i.e. `z64actor.h`)